.mocharc.yaml

```diff
@@ -0,0 +1,4 @@
+require:
+  - ts-node/register 
+  - global-jsdom/register 
```

Comment:
is it required for ArcxAnalyticsProvider tests?

Reply:
yes